riotfile.py

```diff
@@ -1154,6 +1154,26 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="pymysql",
+            command="pytest {cmdargs} tests/contrib/pymysql",
+            venvs=[
+                Venv(
+                    pys=select_pys(min_version="3.6"),
+                    pkgs={
+                        "pymysql": [
+                            "~=0.7",
+                            "~=0.8",
+                            "~=0.9",
+                            "~=0.7",
+                            "~=1.0.1",
+                            "~=1.0.2",
```

Comment:
```suggestion
                            "~=1.0.2",
```

those will install the same version. we usually do each minor version, not each patch version